.github/workflows/test.yml

```diff
@@ -101,6 +101,7 @@ jobs:
           APP_KEY=base64:vHI+YHgkyCDad31iPEErGSNEOWO21wNzV+zyENKQv04=
           APP_URL=http://127.0.0.1:8000
           APP_ENV=testing
+          APP_DEBUG=true
```

Comment:
Why are you setting APP_DEBUG=true?

Reply:
Ok, once we are ready, we can remove APP_DEBUG=true and the check for testing env in Error Reporting.